core/src/avm1/activation.rs

```diff
@@ -2305,16 +2305,23 @@ impl<'a, 'gc> Activation<'a, 'gc> {
         action: WaitForFrame,
         r: &mut Reader<'_>,
     ) -> Result<FrameControl<'gc>, Error<'gc>> {
-        let loaded = self
-            .target_clip()
-            .and_then(|dobj| dobj.as_movie_clip())
-            .map(|mc| mc.frames_loaded() > min(action.frame, mc.total_frames() - 1))
-            .unwrap_or(true);
+        let frame_num = action.frame;
+        let loaded = if frame_num > 16000 {
```

Comment:
I guess it would make sense to define this value as a constant. But not sure what could be best place for this.